nexus/lib/pwscf_analyzer.py

```diff
@@ -930,6 +868,85 @@ def plot_bandstructure(self, filename=None, filepath=None, max_min_e = None, sho
             show()
         #end if
     #end def plot_bandstructure
+    def md_statistics(self,equil=None,autocorr=None):
```

Comment:
`md_statistics` and `md_plots` appear to be repeated with some variations in a few files.  Is there a good way to consolidate and avoid duplication?